frontend/public/components/utils/request-size-input.tsx

```diff
@@ -19,7 +19,7 @@ export class RequestSizeInput extends React.Component<RequestSizeInputProps> {
   };
 
   render() {
-    const { describedBy, name } = this.props;
+    const { describedBy, name, inputId } = this.props;
```

Comment:
By console naming convention this should be `inputID`

Reply:
Ok, thanks! Updated it 🙂